src/black/__init__.py

```diff
@@ -300,6 +306,26 @@ def validate_regex(
         " functionality in the next major release."
     ),
 )
+@click.option(
+    "--unstable",
+    is_flag=True,
+    help=(
+        "Enable potentially disruptive style changes that have known bugs or are not"
+        " currently expected to make it into the stable style Black's next major"
+        " release. Implies --preview."
+    ),
+)
+@click.option(
+    "--enable-unstable-feature",
+    type=click.Choice([v.name for v in UNSTABLE_FEATURES]),
```

Comment:
It could be worth it to be more permissive here, so folks can set `--enable-unstable-feature hug_parens` and forget about it for a year. But I think that much bother is probably fine for preview...

Reply:
I think that's actually fine since we only allow `--enable-unstable-feature` if you also have `--preview`, so setting it for a flag that's in preview is a no-op. I'll change this.